sdk/cosmos/azure-cosmos-spark_3-0_2-12/docs/quick-start.md

```diff
@@ -0,0 +1,132 @@
+
+[//]: # (//TODO: moderakh add maven coordinates once published)
+
+The getting started guide is based on PySpark however you can use the equivalent scala version as well.
+You can run the following code snippet in an Azure Databricks PySpark notebook.
+
+### Create Cosmos DB Database and Container
+
+Set Cosmos DB account credentials, and the Cosmos DB Database name and container name.
+```python
+cosmosEndpoint = "https://REPLACEME.documents.azure.com:443/"
+cosmosMasterKey = "REPLACEME"
+cosmosDatabaseName = "sampleDB"
+cosmosContainerName = "sampleContainer"
+
+cfg = {
+  "spark.cosmos.accountEndpoint" : cosmosEndpoint,
+  "spark.cosmos.accountKey" : cosmosMasterKey,
+  "spark.cosmos.database" : cosmosDatabaseName,
+  "spark.cosmos.container" : cosmosContainerName,
+}
+```
+
+see [General Configuration](./configuration-reference.md#Generic Configuration) for more detail.
+
+You can use the new Catalog API to create a Cosmos DB Database and Container through Spark.
+Configure Catalog Api to be used
+```python
+# Configure Catalog Api to be used
+spark.conf.set("spark.sql.catalog.cosmosCatalog", "com.azure.cosmos.spark.CosmosCatalog")
+spark.conf.set("spark.sql.catalog.cosmosCatalog.spark.cosmos.accountEndpoint", cosmosEndpoint)
+spark.conf.set("spark.sql.catalog.cosmosCatalog.spark.cosmos.accountKey", cosmosMasterKey)
+```
+
+Create a Cosmos DB database
+```python
+# create a cosmos database using catalog api
+spark.sql("CREATE DATABASE IF NOT EXISTS cosmosCatalog.{};".format(cosmosDatabaseName))
+```
+
+Create a Cosmos DB container:
+```python
+# create a cosmos container using catalog api
+spark.sql("CREATE TABLE IF NOT EXISTS cosmosCatalog.{}.{} using cosmos.items TBLPROPERTIES(partitionKeyPath = '/id', manualThroughput = '1100')".format(cosmosDatabaseName, cosmosContainerName))
+```
+Cosmos Catalog API for creating container supports setting throughput and partition-key-path for the container to be created.
+
+see [Catalog API](./catalog-api.md) for more detail.
+
+### Ingest Data to Cosmos DB
+
+The name of the Cosmos DB Data Source is "cosmos.items". following shows how you can write a memory dataframe consisting of two items to Cosmos DB.
+```python
+# Ingest data to Cosmos DB
+spark.createDataFrame((("cat-alive", "Schrodinger cat", 2, True), ("cat-dead", "Schrodinger cat", 2, False)))\
+  .toDF("id","name","age","isAlive") \
+   .write\
+   .format("cosmos.items")\
+   .options(**cfg)\
+   .mode("APPEND")\
+   .save()
+```
+Note that `id` is a mandatory field for Cosmos DB.
+
+see [Write Configuration](./configuration-reference.md#write-config) for more detail.
+
+
+### Query Cosmos DB
+
+```python
+# Query data from Cosmos DB
+from pyspark.sql.functions import col
+
+df = spark.read.format("cosmos.items").options(**cfg)\
+ .option("spark.cosmos.read.inferSchemaEnabled", "true")\
+ .load()
+
+df.filter(col("isAlive") == True)\
+ .show()
+```
+
+see [Query Configuration](./configuration-reference.md#query-config) for more detail.
+
```

Comment:
perhaps, helpful to include the spark / hive table creation snippet (pointing to the cosmosdb container), if & when we start supporting it?

Reply:
Thanks @evinjchalil good suggestion. will track here
https://github.com/Azure/azure-sdk-for-java/issues/19629